std/digest/md.d

```diff
@@ -179,7 +179,7 @@ struct MD5
             S44 = 21,
         }
 
-        private void transform(const(ubyte[64])* block) pure nothrow @nogc
```

Comment:
I actually think there's merit in using private like this as opposed to the label form. In the label form the user has to browse in this case all the way up to line 98 to figure that the symbol is indeed private. And of course make sure there's no other intervening label. In the hierarchy of ease to assess, we have per-symbol (nicest), scope-introducing, and label.

Can you please change this PR a bit to only keep the label removals?